src/main/java/sirius/biz/analytics/events/EventRecorder.java

```diff
@@ -226,6 +227,69 @@ public <E extends Event> int countEventsInRange(Class<E> eventType,
         });
     }
 
+    /**
+     * Sums the number of events for the provided column which have occurred based on the given <tt>queryTuner</tt>.
```

Comment:
It does not sum the number of events, but the values in the given field of those events, right?

```suggestion
     * Sums the values for the provided column of events which have occurred based on the given <tt>queryTuner</tt>.
```